README.md

```diff
@@ -9,16 +9,38 @@
 
 ## Overview
 
-HealthyHome is an open-source smart home device which monitors the indoors environment conditions, based on a Raspberry Pi.
+HealthyHome is an open-source project centred around a smart home device which monitors the indoor environment conditions of a home.
+
+### Objectives
+
+-   Allow a user to characterise important parameters of their indoor home environment
+-   Provide easy access to both real-time and historical versions of this information
+-   Allow interested parties to recreate this project
```

Comment:
Redundant as it is an open-source project

Reply:
Removed lines 16 and 18.